bin/cli.js

```diff
@@ -41,19 +41,30 @@ if (argv.config) {
     config.files = [config.files];
   }
   files = config.files;
+
+  //Set ignore param
+  if (typeof config.ignore === 'string') {
+    config.ignore = [config.ignore];
+  }
+  ignore = config.ignore;
 }
 
 //Get from/to parameters from CLI args if not defined in config file
 if (typeof from === 'undefined') {
-  from = argv._.shift();
+  from = argv._[0];
 }
 if (typeof to === 'undefined') {
-  to = argv._.shift();
+  to = argv._[1];
 }
 
 //Get files
 if (!files) {
-  files = argv._;
+  files = argv._[2];
```

Comment:
@adamreisnz The only issue I see with having `ignore` be a flag instead of a parameter is that the glob's don't automatically get expanded in flags.